crates/l2/proposer/l1_watcher.rs

```diff
@@ -47,14 +66,32 @@ impl L1Watcher {
             eth_client: EthClient::new_from_config(eth_config),
             address: watcher_config.bridge_address,
             topics: watcher_config.topics,
-            check_interval: Duration::from_millis(watcher_config.check_interval_ms),
             max_block_step: watcher_config.max_block_step,
             last_block_fetched: U256::zero(),
             l2_proposer_pk: watcher_config.l2_proposer_private_key,
-            l2_proposer_address: watcher_config.l2_proposer_address,
```

Comment:
I think we are not using this variable. Should we remove the variable from the `watcher_config` and delete the env variable in the `.env.example`?

Reply:
Removed them here [a61b9f5](https://github.com/lambdaclass/ethrex/pull/1205/commits/a61b9f50008c773c503ba8e6f7a14f2afb3f3a72)